vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/logcat/parse/LogcatEventsAdapter.kt

```diff
@@ -40,6 +40,10 @@ class LogcatEventsAdapter(private val parsedEventsListener: LogcatEventsListener
 
     private fun LogcatMessage.parseZygoteExitReason(device: AndroidDevice): FatalError? {
         if (tag == ZYGOTE_TAG) {
+            if (body.contains(ZYGOTE_KILLED_EXTERNALLY_EXIT_REASON)) {
```

Comment:
Maybe add test to check that null will be returned? Or you can skip it if it will take a lot of time